R/epi_recipe.R

```diff
@@ -397,3 +397,77 @@ kill_levels <- function(x, keys) {
   for (i in which(names(x) %in% keys)) x[[i]] <- list(values = NA, ordered = NA)
   x
 }
+
+#' @export
```

Comment:
This probably shouldn't be exported. But we also want `print.epi_recipe()` which _should_ be exported. Currently, because an `epi_recipe` is a subclass of `recipe`, calling `print(epi_recipe)` will dispatch to `print.recipe()`. If we want numbered, then we need to make a method.